cmd/execution_builder.go

```diff
@@ -739,6 +747,71 @@ func (exeNode *ExecutionNode) LoadStopControl(
 	return stopControl, nil
 }
 
+func (exeNode *ExecutionNode) LoadRegisterStore(
+	node *NodeConfig,
+) error {
+	if !exeNode.exeConf.enableStorehouse {
+		node.Logger.Info().Msg("register store disabled")
+		return nil
+	}
+
+	node.Logger.Info().Msgf("register store enabled, open pebble db at: %v", exeNode.exeConf.registerDir)
```

Comment:
I would prefer something like this.
I find it easier to consume later when you are looking for this in grafana.

```suggestion
	node.Logger.Info().
		String("pebble_db_path": exeNode.exeConf.registerDir).
		Msgf("register store enabled") 
```